lib/parzival_web/live/app/jobs/offer_live/show.html.heex

```diff
@@ -77,11 +77,21 @@
           <h2 class="text-lg font-medium text-gray-900">About the Company</h2>
           <%= live_redirect to: Routes.company_show_path(@socket, :show, @offer.company), class: "mt-4 flex border border-gray-300 shadow-sm hover:bg-gray-50 px-6 py-6 rounded-md items-center" do %>
             <div class="flex-shrink-0 mr-4 md:mb-4">
-              <span class="inline-flex justify-center items-center w-16 h-16 rounded-full md:w-24 md:h-24 bg-secondary">
-                <span class="text-3xl font-medium leading-none text-white md:text-5xl">
-                  <%= extract_initials(@offer.company.name) %>
-                </span>
-              </span>
+              <div>
+                <%= if is_nil(Parzival.Uploaders.ProfilePicture.url({@offer.company.picture, @offer.company}, :medium)) do %>
```

Comment:
```suggestion
                <%= if is_nil(@offer.company.picture) do %>
```

Shouldn't we just test if the field is null because we don't want to provide a default value?